app/ping/judge_endpoint/endpoint.py

```diff
@@ -0,0 +1,27 @@
+""" Endpoint for judge's data input """
+from flask import Blueprint, json, request
+from .endpoint_check import Endpoint_check
+from app.executor.executor import Executor
+
+
+judge_endpoint_bp = Blueprint("judge_endpoint", __name__)
+
+
+@judge_endpoint_bp.route("/judge/end", methods=["GET", "POST"])
+def judge_end() -> (json, int):
+    """Endpoint receiving data from back-end
+    return: json containing either a error message or the data returned by the executor, and a response status
+    """
+    check = Endpoint_check(request)
+
+    if not check.judge_data_complete():
```

Comment:
Habría que analizar como se junta esto con el middleware como tal, para que no se valide lo mismo dos veces

Reply:
Hasta el momento desde EndpointCheck se está corriendo el método del middleware y revisando/ajustando otras cositas. Si no es necesario o si se fuera a ejecutar los métodos desde otra parte, es quitar 

`self.middleware = Middleware(data)

        if not self.middleware.validate_code():
            self.response = {
                "message": "Wrong values. Programming language not standardized, code is empty or time limit is unreasonable"
            }
            return False`

de endpoint_check.py